apps/roboshield/src/pages/about.tsx

```diff
@@ -0,0 +1,26 @@
+import { getPageServerSideProps } from "@/roboshield/lib/data";
+import BlockRenderer from "@/roboshield/components/BlockRenderer";
+import { Page } from "@/root/payload-types";
+
+type BlockType = Page["blocks"];
+
+interface PageProps {
+  blocks: BlockType;
+}
```

Comment:
Shouldn't `PageProps` or `ServerSideProps` be defined once in lib/data and reused? I'm assuming all pages will have similar structure.